.gitignore

```diff
@@ -22,3 +22,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+
+# vs code
```

Comment:
please add .vscode/ to .gitignore

Reply:
please also remove the currently committed .vscode/ folder -> there are ways to do so AFTER you have updated the .gitignore, let me know if you need help